libs/barista-components/radial-chart/src/radial-chart.ts

```diff
@@ -174,37 +211,39 @@ export class DtRadialChart implements AfterContentInit, OnDestroy {
     private _platform: Platform,
     // TODO: remove this sanitizer when ivy is no longer opt out
     private _sanitizer: DomSanitizer,
+    private _resizer: DtViewportResizer,
```

Comment:
Non optional addition in the constructor parameters is considered a breaking change. 
As we do not need to clamp the svg sizes, I guess we can remove this one again and it will not block the progress of this one until barista version 8.

Reply:
@tomheller I'll remove the clamping butI think I prefer to wait for barista 8 and include resizing of the chart. Before it was stuck with the initial dimensions

Reply:
The svg should have resized with the container. I'll have to check on monday if resizing is broken on the radial chart. Thanks for pointing this out.

Reply:
Resizing of the radial chart seems to work as expected on master. I don't think we need the viewPortResizer. 

![May-25-2020 08-52-57](https://user-images.githubusercontent.com/6338434/82786578-2d78d300-9e65-11ea-96f9-08058abaaefc.gif)

Reply:
Well, yes and no, it scales but it does not keep the width of the donut as it should be. But I'll remove it and review it in the future if need be

Reply:
yes, you are right. it scales the whole svg, which should be fine for now. 
With the viewport resizer, it would be possible to scale the donut-thickness acordingly